crates/sui-core/src/authority_active.rs

```diff
@@ -203,25 +204,17 @@ impl<A> ActiveAuthority<A>
 where
     A: AuthorityAPI + Send + Sync + 'static + Clone,
 {
-    pub async fn spawn_all_active_processes(self) {
-        self.spawn_active_processes(true, Some(CheckpointProcessControl::default()))
+    pub async fn spawn_checkpoint_process(self) {
+        self._spawn_checkpoint_process(Some(CheckpointProcessControl::default()))
             .await
     }
 
     /// Spawn all active tasks.
-    pub async fn spawn_active_processes(
+    pub async fn _spawn_checkpoint_process(
```

Comment:
Lets call this `_with_config` instead of underscore at the start.

Reply:
This seems unaddressed

Reply:
Ah my bad. I was looking at an out-of-dated view.